conf/evolutions/default/86.sql

```diff
@@ -0,0 +1,9 @@
+# --- !Ups
+
+ALTER TABLE `patients`
+ADD COLUMN `isDeleted` DATETIME NULL DEFAULT NULL AFTER `photo_id`;
+
+# --- !Downs
+
+ALTER TABLE `patients`;
+DROP COLUMN `isDeleted;
```

Comment:
Just tried to execute the down script - 'isDeleted' needs a closing grave accent otherwise it won't work.